charts/sptribs-case-api/values.preview.template.yaml

```diff
@@ -6,8 +6,8 @@ java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
   environment:
-    CASE_DATA_STORE_BASEURL: http://{{ .Release.Name }}-ccd-data-store-api
-    CASE_DOCUMENT_AM_BASEURL: http://ccd-case-document-am-api-aat.service.core-compute-aat.internal
+    CASE_DATA_STORE_BASEURL: http://ccd-data-store-api-sptribs-case-api-pr-${CHANGE_ID}.service.core-compute-preview.internal
```

Comment:
I would suggest that for your chart file your preview chart file you go with one naming convention only for those calls you know are _preview_ to _preview_.

so: hopefully 

```
CASE_DATA_STORE_BASEURL: http://${SERVICE_NAME}-ccd-data-store-api
```

or

```
CASE_DATA_STORE_BASEURL: http://sptribs-case-api-pr-${CHANGE_ID}-ccd-data-store-api
```

(I think/hope the above are both equivalent: so pick the one that is most readable to your team).

and follow the same throughout for all that you know are preview-preview connections.  (i.e. anything that is not already configured as AAT). and look to leave out: `.service.core-compute-preview.internal` until you are configuring the FTA scripts inside jenkins config.